webapi/src/main/scala/org/knora/webapi/messages/v2/responder/resourcemessages/ResourceMessagesV2.scala

```diff
@@ -606,15 +605,22 @@ object CreateResourceRequestV2 extends KnoraJsonLDRequestReaderV2[CreateResource
                                     settings = settings,
                                     log = log
                                 )
+
                                 maybeCustomValueIri: Option[SmartIri] = valueJsonLDObject.maybeIDAsKnoraDataIri
-                                maybeCustomValueUUID: Option[UUID] = valueJsonLDObject.maybeUUID
+                                maybeCustomValueUUID: Option[UUID] = valueJsonLDObject.maybeUUID(OntologyConstants.KnoraApiV2Complex.ValueHasUUID)
 
-                                // Get the values's creation date.
+                                // Get the value's creation date.
+                                // TODO: creationDate for values is a bug, and will not be supported in future. Use valueCreationDate instead.
```

Comment:
I guess you can remove this comment and TODO from here. As far as I see, you have fixed it already.

Reply:
The way I wrote it, it will accept both. The TODO is to remove support for creationDate later, leaving only the support for valueCreationDate. I wrote it this way because I thought we weren't allowed to make breaking changes anymore. :) Do you think that's right, or should I should just make it a breaking change?

Reply:
> The way I wrote it, it will accept both. The TODO is to remove support for creationDate later, leaving only the support for valueCreationDate. I wrote it this way because I thought we weren't allowed to make breaking changes anymore. :) Do you think that's right, or should I just make it a breaking change?

ooh, tough question. I believe we must be able to do breaking changes, but in agreement with front-end/JS-lib team so that they can immediately adapt their code.

Reply:
OK then let's leave it like this, as a deprecation, to give them time to adapt. Then I'll make the breaking change later.